torch/csrc/Module.cpp

```diff
@@ -174,6 +174,11 @@ static PyObject * THPModule_setNumThreads(PyObject *module, PyObject *arg)
   Py_RETURN_NONE;
 }
 
```

Comment:
I think we have a better way to create a binding, and this one is deprecated, we moved to pybind, example of how to add a binding in one line of code: https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/init.cpp#L57

Reply:
When I used `pybind11`, `add_docstr` failed because the function already had an (empty) docstring. Obviously I can just define the docstring in `Module.cpp` but then the doc strings aren't all in one place. What do you think?

Reply:
Moved to pybind, can you take another look @ilia-cher?